<launch>
  <!-- args = list of led id -->
  <node pkg="visual_gps" type="visual_gps_2d_node" name="visual_gps_2d" args="0 1 2">
    <param name="map_file" value="$(find visual_gps)/config/FEATURE_FILES/map.txt" />
    <param name="camera_file" value="$(find visual_gps)/config/FEATURE_FILES/camera.txt" />
    <param name="start_frame" value="/world" />
    <param name="end_frame" value="/camera"/>
  </node>
</launch>
